keanu-project/src/main/java/io/improbable/keanu/vertices/SimpleVertexDictionary.java

```diff
@@ -0,0 +1,30 @@
+package io.improbable.keanu.vertices;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import io.improbable.keanu.tensor.Tensor;
+
+public class SimpleVertexDictionary implements VertexDictionary {
+
+    private final Map<VertexLabel, Vertex<?>> map;
```

Comment:
`map`? Is there a more descriptive name?

Reply:
Fair point 😆 Would you prefer `dictionary`? Still pretty generic but I think that's inevitable.